plugins/network-device-injector/README.md

```diff
@@ -0,0 +1,80 @@
+## Network Device Injector Plugin
+
+This sample plugin can inject existing network devices into containers using pod annotations.
```

Comment:
status .. work in progress? Maybe mention limitations.. such as pod networking not host .. and if it should work for usernamespace isolated pods..   (wondering if there will be any non host runtime handler issues e.g. kata/cc)

Maybe mention, link, the on-going work for multi-network, network device claims that would naturally provide a built in means to inject.